Automattic/sensei
# Add Sensei block spacings

### Changes proposed in this Pull Request

* This PR purpose is to make sure the spacing between blocks will work properly in the main themes.
* Basically it introduces a wrapper class (`sensei-block-wrapper`) to the Sensei blocks, adding this space. This space is added in the same element that some themes add the spacing, so it also prevents the duplicated spacing.
* Since it's introduced by the same class to all the Sensei blocks, it's easy to remove in the future, if we want, depending on how the themes improve.

### Testing instructions

* Create a new course with all blocks.
* Change between the main themes.
* Visit the course page with every tested theme, with the user enrolled and not enrolled, and make sure the spacing between the blocks is working properly.
* Test with the [`optional_templates` feature flag](https://github.com/Automattic/sensei/pull/3706) enabled and disabled to make sure it's working regardless of our decision about this.

I tested it with the themes: Divi, Astra, Twenty Twenty, Storefront, and Aardvark.


### Screenshot / Video

<img width="643" alt="Screen Shot 2020-11-05 at 16 28 07" src="https://user-images.githubusercontent.com/876340/98287632-5eb50d80-1f84-11eb-882f-39d1df67459f.png">

---

<img width="642" alt="Screen Shot 2020-11-05 at 16 28 18" src="https://user-images.githubusercontent.com/876340/98287639-61affe00-1f84-11eb-83d7-024f8efe41bb.png">

---

<img width="633" alt="Screen Shot 2020-11-05 at 16 28 29" src="https://user-images.githubusercontent.com/876340/98287643-62e12b00-1f84-11eb-8bda-4492028a125b.png">
